Makefile

```
@@ -173,3 +173,49 @@ server-profile-cpu:
 jaeger:
 	open http://localhost:16686
 	docker run --rm -p 5775:5775/udp -p 6831:6831/udp -p 6832:6832/udp -p 5778:5778 -p 16686:16686 -p 14268:14268 -p 9411:9411 -e COLLECTOR_ZIPKIN_HTTP_PORT=9411 jaegertracing/all-in-one:1.7
+
+e2e-setup:
+	make e2e-setup-git
+	make e2e-setup-kind
+	make e2e-setup-fluxd
```

Any reason for not just running these as make pre-targets?

```suggestion
e2e-setup: e2e-setup-git e2e-setup-kind e2e-setup-fluxd
```

---

does this run before "body" of the target and sequentially?

---

Yes.